doc/opencv-logo.md

```diff
@@ -0,0 +1,7 @@
+OpenCV logo has been originally designed and contributed to OpenCV by Adi Shavit in 2006. The graphical part consists of three stylized letters O, C, V, colored in the primary R, G, B color components, used by humans and computers to perceive the world. It is shaped in a way to mimic the famous [Kanizsa's triangle](https://en.wikipedia.org/wiki/Illusory_contours) to emphasize that the prior knowledge and internal processing are at least as important as the actually acquired "raw" data.
+
+The restyled version of the logo has been designed and contributed by [xperience.ai](xperience.ai) in July 2020 for the [20th anniversary](https://opencv.org/anniversary/) of OpenCV.
```

Comment:
Would you like to provide a valid link?

Reply:
Hi @dkurt! You mean full link like https://xperience.ai/?

Reply:
Yes, because if you will try to click at https://github.com/AnnaPetrovicheva/opencv/blob/logo_text/doc/opencv-logo.md - you will see "Page not found" GitHub's error

Reply:
Done!